### PR TITLE
initial user's guide

### PR DESCRIPTION
**Pull request by jmazzitelli**
This is [the main file to review](https://github.com/kiali/openshift-servicemesh-plugin/blob/0c51b73bb626b53efddffddb061db2c0e0169f46/docs/users-guide.md)

### CODE REVIEW

**Comment by jmazzitelli** [comment]
I'm going to open this up for review and comment. This is a very basic user's guide that describes the main screens, albeit briefly. This currently does not have alot of detail explaining all the different features and functions of the topology graph and the workload subtabs. Alot more can be added to provide more details of all the different options and features in these screens. But for now I took screenshots of the main screens and just gave basic overview. We can either iterate in this PR to add more details or we can merge this and iterate with future PRs (assuming this PR has the basic outline of the user docs that we want). 

I am not a tech writer so... keep that in mind. :)

File: docs/users-guide.md [hunk 1]
```diff
@@ -1,3 +1,78 @@
 # OpenShift Service Mesh Console User's Guide
 
-TODO
+The OpenShift Service Mesh Console (aka OSSMC) is an extension to the OpenShift Console which provides visibility into your Service Mesh. The functionality you see described here is very similar to that of the standalone Kiali Console (see https://kiali.io). In fact, you can still access the standalone Kiali Console if you wish. This User's Guide, however, will discuss the functionality that you see from within the OpenShift Console itself.
+
+:warning: The OSSMC [only supports a single tenant today](https://github.com/kiali/openshift-servicemesh-plugin/issues/28). Whether that tenant is configured to access only a subset of OpenShift projects or has access cluster-wide to all projects does not matter, however, only a single tenant can be accessed.
+
+# Overview
+
+The _Overview_ page provides a summary of your mesh by showing cards representing the namespaces participating in the mesh. Each namespace card has summary metric graphs and additional health details. There are links in the cards that take you to other pages within OSSMC.
```
Comment (nrfox) [inline docs/users-guide.md:9]: imo mentions of the tab name should be bold and not italicized.

File: docs/users-guide.md [hunk 2]
```diff
@@ -1,3 +1,78 @@
 # OpenShift Service Mesh Console User's Guide
 
-TODO
+The OpenShift Service Mesh Console (aka OSSMC) is an extension to the OpenShift Console which provides visibility into your Service Mesh. The functionality you see described here is very similar to that of the standalone Kiali Console (see https://kiali.io). In fact, you can still access the standalone Kiali Console if you wish. This User's Guide, however, will discuss the functionality that you see from within the OpenShift Console itself.
```
Comment (nrfox) [inline docs/users-guide.md:3]: It'd be helpful adding a blurb here saying that the plugin both provides its own "tab" (or whatever the left side nav section is called) and enhances existing ones like the "Workloads" pages in order to set expectations on what the plugin does.

File: docs/users-guide.md [hunk 3]
```diff
@@ -1,3 +1,78 @@
...
+
+:warning: The OSSMC [only supports a single tenant today](https://github.com/kiali/openshift-servicemesh-plugin/issues/28). Whether that tenant is configured to access only a subset of OpenShift projects or has access cluster-wide to all projects does not matter, however, only a single tenant can be accessed.
+
+# Overview
```
Comment (nrfox) [inline docs/users-guide.md:7]: Do you need a screenshot for the overview page as well?

File: docs/users-guide.md (hunk 1 above)
Comment (jmazzitelli) [reply docs/users-guide.md:9]: I don't know the standard doc way of doing it. Perhaps @JStickler can tell us.

File: docs/users-guide.md (hunk 3 above)
Comment (jmazzitelli) [reply docs/users-guide.md:7]: Screenshot of the overview page is the first one in the user's guide: https://github.com/kiali/openshift-servicemesh-plugin/blob/eed0a8eab662ab476adc265d22bcee4e12ba315c/docs/users-guide.md#overview

This: https://github.com/kiali/openshift-servicemesh-plugin/blob/eed0a8eab662ab476adc265d22bcee4e12ba315c/docs/20-overview.png

File: docs/users-guide.md (hunk 2 above)
Comment (jmazzitelli) [reply docs/users-guide.md]: done.

File: docs/users-guide.md [hunk 4]
```diff
@@ -1,3 +1,78 @@
...
+
+## Workload: Metrics
+
+You can see both inbound and outbound metric graphs in the corresponding sub-tabs. All the workload metrics can be displayed here, providing you with a detail view of the performance of your workload. You can enable the tracing span integration which allows you to see which spans occurred at the same time as the metrics. You can then click on a span marker in the graph to view the specific spans associated with that timeframe.
+
+![Workload: Metrics](26-workload-metrics.png)
+
+## Workload: Traces
+
+The _Traces_ sub-tab provides a graph that illustrates the trace spans that were collected over the given timeframe. Click on a bubble to drill down into those trace spans; the trace spans can provide you the most low-level detail within your workload application, down to the individual request level.
```
Comment (jshaughn) [inline docs/users-guide.md:57]:
```suggestion
The _Traces_ sub-tab provides a chart showing the trace spans collected over the given timeframe. Click on a bubble to drill down into those trace spans; the trace spans can provide you the most low-level detail within your workload application, down to the individual request level.
```

File: docs/users-guide.md [hunk 5]
```diff
@@ -1,3 +1,78 @@
...
+
+![Workload: Traces](27-workload-traces.png)
+
+The trace details view will give further details, including heatmaps that provide you with a comparision of one span in relation to other requests and spans in the same timeframe.
```
Comment (jshaughn) [inline docs/users-guide.md:61]:
```suggestion
The trace details view will give further details, including heatmaps that provide you with a comparison of one span in relation to other requests and spans in the same timeframe.
```

**Review by jshaughn — CHANGES_REQUESTED** [review]
Thanks for putting this together, a few suggestions, hopefully not already out of date as I see there was already an update...

File: docs/users-guide.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (jmazzitelli) [reply docs/users-guide.md:9]: I did add a blurb at the top about this as a suggestion from nick.. see: https://github.com/kiali/openshift-servicemesh-plugin/pull/76#discussion_r963855656 and https://github.com/kiali/openshift-servicemesh-plugin/pull/76/commits/b3d8ee99d58b041e9f507da2610386526c159599

File: docs/users-guide.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jmazzitelli) [reply docs/users-guide.md]: this is done here: https://github.com/kiali/openshift-servicemesh-plugin/pull/76/commits/b320d73b759d2f7ca644979574560aeeca9af2e1

**Review by jshaughn — CHANGES_REQUESTED** [review]
Almost there.

**Comment by jmazzitelli** [comment]
> There are several instances that should go from italics to bold.

Fine.

**Pull request merged**